modules/dag-l0/src/main/scala/io/constellationnetwork/dag/l0/domain/cell/AlgebraCommand.scala

```diff
@@ -12,5 +14,7 @@ object AlgebraCommand {
   case class EnqueueStateChannelSnapshot(snapshot: StateChannelOutput) extends AlgebraCommand
   case class EnqueueDAGL1Data(data: Signed[Block]) extends AlgebraCommand
   case class EnqueueUpdateNodeParameters(data: Signed[UpdateNodeParameters]) extends AlgebraCommand
+  case class EnqueueDelegatedStake(data: DelegatedStakeOutput) extends AlgebraCommand
+  case class EnqueueNodeCollateral(data: NodeCollateralOutput) extends AlgebraCommand
```

Comment:
It might be a lot easier to have both of these types extend some new direct type (similar to the way we treat the data artifact additions in the ML0 for the changes for SpendTransaction / AllowSpend,

so rather than dealing with this queue in multiple places, you could have a single new type here, indicating a direct GL0 embedded artifact. That way, we don't need to modify any of these functions again in the future.